google-cloud-pubsub/samples/acceptance/subscriptions_test.rb

```diff
@@ -217,4 +219,21 @@
       )
     end
   end 
+
+  it "supports creating bigquery subscription" do
+    project_id = pubsub.project
+    topic_id = @topic.name
+    subscription_id = random_subscription_id
+    table_id = create_table 
+
+    assert_output "BigQuery subscription created: #{subscription_id}.\nTable for subscription is: #{table_id}\n" do     
+      pubsub_create_bigquery_subscription(
+        project_id: project_id,
+        topic_id: topic_id,
+        subscription_id: subscription_id,
+        bigquery_table_id: table_id
+      )
+    end
+    cleanup_bq @table, @dataset
```

Comment:
It's okay to use `after do`. Just check whether the instance variables are defined before attempting to clean up. (You should either use `after do` or use an `ensure` block so that the cleanup happens even if the assertion fails.)